README.md

```diff
@@ -25,3 +26,25 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+### Analysis
+
+$T(n) = 3T(n/3)$ because there are 3 recursive calls and the array is divided into 3 </br>
```

Comment:
What about the work to combine the partial results?

Reply:
I think I should have that fixed now. since that step is a constant, I ignored it because  don't think that it would change the final output. I have added +c to the places that need it.